src/components/query/QueryEditor.tsx

```diff
@@ -88,16 +123,16 @@ export class QueryEditor extends PureComponent<Props> {
               width={18}
               options={regions}
               value={standardRegionOptions.find((v) => v.value === query.region) || defaultRegion}
-              onChange={this.onRegionChange}
+              onChange={onRegionChange}
               backspaceRemovesValue={true}
               allowCustomValue={true}
               isClearable={true}
               menuPlacement="bottom"
             />
           </InlineField>
         </div>
-        {this.renderQuery(query)}
-      </>
-    );
-  }
+      
+      {renderQuery(query)}</>}
```

Comment:
Let's take this one more step and make this a functional react component so we can write:
```
<Query query={query} />
```

Or ideally some more descriptive name than that haha

Reply:
This is an old component, so if that's ok, I'll skip this to limit refactoring in this PR, as you mentioned above